tests/foreman/api/test_capsulecontent.py

```diff
@@ -1176,7 +1176,8 @@ def test_positive_sync_CV_to_multiple_LCEs(
         result = module_capsule_configured.nailgun_capsule.content_lifecycle_environments()
         # there can and will be LCEs from other tests and orgs, but len() >= 2
         assert len(result['results']) >= 2
-        assert lce1.id and lce2.id in [capsule_lce['id'] for capsule_lce in result['results']]
+        assert lce1.id
+        assert lce2.id in [capsule_lce['id'] for capsule_lce in result['results']]
```

Comment:
This is formatted correctly or the original test author was mistaken about what they were testing. There are two checks going on here, so it is a separation of them to better clarify where the failure is.

1. check that lce1.id is a truthy value
2. check that lce2.id is in the results list

Also, be careful with your recommended change since it doesn't do what you expect. That first statement will only ever return the value of lce2.id. Consider the following:
```python
>>> (1 and 2)
2
>>> (1 or 2)
1
```

Reply:
Also its not 2 assertions, its one assertion to verify both items are in the list!

Reply:
Hey, all of you are right! :D
Yes, the intention was obviously test that both LCEs are assigned to the Capsule and yes, I was mistaken and `lce1 and lce2` doesn't work as I expected.
So we should rather have something like
```
assert all(item in [capsule_lce['id'] for capsule_lce in result['results']] for item in [lce1.id, lce2.id])
```
Unless I am mistaken again or you have much nicer way to test that.

Reply:
Or
```
assert {lce1.id, lce2.id}.issubset([capsule_lce['id'] for capsule_lce in result['results']])
```
might be more pythonic?